iotedgedev/modules.py

```diff
@@ -65,7 +66,7 @@ def add(self, name, template, group_id):
 
             shutil.move(temp_template_dir, os.path.join(cwd, name))
 
-            module = Module(self.envvars, self.utility, name)
+            module = Module(self.envvars, self.utility, os.path.join(self.envvars.MODULES_PATH, name))
```

Comment:
the MODULES_PATH env only used for c module? others use cwd?

Reply:
C module also works in cwd. The MODULES_PATH is a relative path which points to the modules folder under cwd. Other modules have their own cli to do scaffolding but for c we do the scaffolding manually. That's why you just see this code in C module.

Reply:
Line 37 already assign cwd to MODULE_PATH to I think better to use cwd here instead of env